tower-buffer/src/worker.rs

```diff
@@ -176,11 +176,11 @@ where
                     // Wait for the service to be ready
                     tracing::trace!(
                         resumed = !first,
-                        message = "worker received request; waiting for service readiness"
```

Comment:
These seem like mostly orthogonal changes to this PR?

Reply:
Yeah, the tracing syntax was improved. I can minimize the diff.